checker/run

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+set -ex
+
+cfg_dir=$(dirname "$0")
+
+. "$cfg_dir"/cfg.sh
```

Comment:
Not sure I understand the idea. The two lines above feel like the runner must be inside the same directory of the configuration, which is probably not desired as it'd force every configuration to have a different runner, right?

I'd also suggest simply ".../options" as the filename instead of "cfg.sh".